lib/pages/chat_list/home_screen.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:badges/badges.dart';
+import 'package:fluffychat/config/app_config.dart';
+import 'package:fluffychat/pages/chat_list/bottom_tab_bar.dart';  
+import 'package:flutter/material.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:vrouter/vrouter.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+
+class HomeScreen extends StatefulWidget {
+
+  final Widget? child;
+
+  const HomeScreen({super.key, this.child});
+
+  @override
+  State<HomeScreen> createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> {
+  int selectedIndex = 1;
+
+  void onDestinationSelected(int selected) {
+    setState(() {
+      selectedIndex = selected;
+      VRouter.of(context).to(BottomTabbar.fromIndex(selected).path);
+      if (selected == BottomTabbar.stories.tabIndex) {
+        Fluttertoast.showToast(
+          msg: "Stories is not ready yet!",
```

Comment:
Why are we not using an international string for this ?

Reply:
IMO, it is temporary, no need to use i10n